android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -514,11 +506,26 @@ private void doInitializePlayer(boolean force) {
 
             showOverlay();
 
-            SourceBuilder sourceBuilder = new SourceBuilder(src.getUrl(), src.getId())
-                    .setIsLive(isLive)
-                    .setMuxData(src.getMuxData(), exoDorisPlayerView.getVideoSurfaceView())
+            SourceBuilder sourceBuilder = new SourceBuilder()
+                    .setUrl(src.getUrl())
+                    .setId(src.getId())
+                    .setMuxProperties(src.getMuxData(), exoDorisPlayerView.getVideoSurfaceView())
                     .setTextTracks(src.getTextTracks())
-                    .setMaxVideoSize(viewWidth, viewHeight);
+                    .setMaxVideoSize(viewWidth, viewHeight)
+                    .setDrmParams(actionToken);
+
+            if (imaDaiSrc != null) {
```

Comment:
```suggestion
            if (isImaDaiStream) {
```

Will this work here? I think it'll improve readability slightly